auslib/web/public/base.py

```diff
@@ -1,19 +1,22 @@
 import cgi
+import connexion
 import logging
 log = logging.getLogger(__name__)
 
-from flask import Flask, make_response, send_from_directory, abort, Response
+from flask import make_response, send_from_directory, abort, Response
 
 from raven.contrib.flask import Sentry
 
 from auslib.AUS import AUS
 from auslib.dockerflow import create_dockerflow_endpoints
 
-app = Flask(__name__)
+connexion_app = connexion.App(__name__)
+connexion_app.add_api('api.yml')
+app = connexion_app.app
```

Comment:
I'm a little surprised that Connexion forces you to use its App instead of providing a middleware, but this is OK.